public/app/features/dashboard-scene/scene/VariableControls.tsx

```diff
@@ -24,14 +25,33 @@ interface VariableSelectProps {
 
 export function VariableValueSelectWrapper({ variable }: VariableSelectProps) {
   const state = useSceneObjectState<SceneVariableState>(variable, { shouldActivateOrKeepAlive: true });
+  const { isSelected, onSelect, isSelectable } = useElementSelection(variable.state.key);
+  const styles = useStyles2(getStyles);
 
   if (state.hide === VariableHide.hideVariable) {
     return null;
   }
 
+  const onPointerDown = (evt: React.PointerEvent) => {
+    if (isSelectable && onSelect) {
+      evt.stopPropagation();
+      onSelect(evt);
+    }
+  };
```

Comment:
When a variable has the label hidden, it's impossible to select it for edit:

https://github.com/user-attachments/assets/1e995cfa-46f1-4cf7-861d-1f3ccbba9cbb

Reply:
Since the variables are quite specific in dashboards edit as they allow value change, maybe the UX for editing should be slightly differen. Instead of having the element itself being selectable, there should be a dotted outline (as it is right now) and a small pop over edit button that only activates the selection, when clicked on it?